packages/cli/src/commands/init/__tests__/editTemplate.test.js

```diff
@@ -74,6 +82,4 @@ test('should edit template', () => {
   expect(
     snapshotDiff(fixtureTree, transformedTree, {contextLines: 1}),
   ).toMatchSnapshot();
-
-  fs.removeSync(testPath);
```

Comment:
FYI @Esemesek it's not safe to do so, because cleanup fn won't be called after failure